pages/allForms/ControlledSubstanceContract.tsx

```diff
@@ -4,11 +4,10 @@ import DateInput from '../../components/userInput/DateInput'
 import CustomCheckBoxField from '../../components/formComponents/CustomCheckBoxField'
 import Signature from '../../components/formComponents/Signature'
 import MainButton from '../../components/Buttons/MainButton'
-import Header from '../../components/navigation/Header'
 import { savePatentForms } from '../../firebase/firebase'
 import { useRouter } from 'next/router'
 import { CircularButton } from '../../components/Buttons/CircularButtonIcon'
-import { ArrowDownLeftIcon, ArrowLeftIcon } from '@heroicons/react/24/outline'
+import { ArrowLeftIcon } from '@heroicons/react/24/outline'
```

Comment:
these changes are in the wrong branch, for feature branch work flow only make changes for that story